scripts/search_test.py

```diff
@@ -93,11 +93,12 @@ def insert_char_at(s, char_to_insert, index):
         assert sz.levenshtein(a, b, 200) == i + 1
 
 
-@pytest.mark.repeat(1000)
-def test_levenshtein_randos():
-    a = get_random_string(length=20)
-    b = get_random_string(length=20)
-    assert sz.levenshtein(a, b, 200) == levenshtein(a, b)
+@pytest.mark.parametrize("reps", [1])
```

Comment:
There is a `pytest-repeat` extension, that adds the decorator. The canonical way would be to use it, and also reset the seed in the main body of the test on every invocation.

Reply:
What's your opinion on `typedef struct` in `stringzilla.h` for `sz_u64_vec_t` and similar constructs. Some C developers are [against them](https://stackoverflow.com/a/4566358), assuming the global namespace gets polluted, but the current implementation doesn't use too many structures and should probably be fine.